iquip/apps/builder.py

```diff
@@ -587,3 +604,33 @@ def onSubmitted(self, rid: int):
     def frames(self) -> Tuple[Tuple[str, BuilderFrame]]:
         """Overridden."""
         return (("", self.builderFrame),)
+
+class _ScanEntry(QWidget):
+    """Entry class for a Scannable argument.
```

Comment:
Is there any better expression instead of "Scannable argument"?

Based on [ARTIQ manual](https://m-labs.hk/artiq/manual/core_language_reference.html?highlight=scan#module-artiq.language.scan), "Scannable object" is one of the good candidates.